app/models/preprint.js

```diff
@@ -1,17 +1,37 @@
 import DS from 'ember-data';
+import OsfModel from 'ember-osf/models/osf-model';
 
-export default DS.Model.extend({
+/**
+ * @module ember-preprints
+ * @submodule models
+ */
+
+/**
+ * Model for OSF APIv2 preprints. This model may be used with one of several API endpoints. It may be queried directly,
+ *  or accessed via relationship fields.
+ * For field and usage information, see:
+ * https://api.osf.io/v2/docs/#!/v2/Preprint_List_GET
+ * https://api.osf.io/v2/docs/#!/v2/Preprint_Detail_GET
+ * https://api.osf.io/v2/docs/#!/v2/User_Preprints_GET
+ * @class Preprint
+ */
+export default OsfModel.extend({
     title: DS.attr('string'),
-    authors: DS.attr(),
-    date: DS.attr(),
-    subject: DS.attr(),
-    abstract: DS.attr(),
-    publisher: DS.attr(),
-    project: DS.attr(),
-    supplementalMaterials: DS.attr(),
-    figures: DS.attr(),
-    license: DS.attr(),
-    path: DS.attr(),
+    // TODO: May be a relationship in the future pending APIv2 changes
+    subjects: DS.attr(),
+    provider: DS.attr('string'),
+    date_created: DS.attr('date'),
+    date_modified: DS.attr('date'),
+    abstract: DS.attr('string'),
     tags: DS.attr(),
-    doi: DS.attr(),
```

Comment:
Interesting! Where is the field defined for the API?

https://github.com/abought/osf.io/blob/99ac101be6a94fd5950ff9c0c317922ec1c12c6d/api/preprints/serializers.py#L28-L28